pkg/ebpf/event_common.go

```diff
@@ -97,8 +101,8 @@ type ConnectionStats struct {
 }
 
 func (c ConnectionStats) String() string {
-	return fmt.Sprintf(
-		"[%s] [PID: %d] [%v:%d ⇄ %v:%d] (%s) %d bytes sent (+%d), %d bytes received (+%d), %d retransmits (+%d)",
+	str := fmt.Sprintf(
+		"[%s] [PID: %d] [%v:%d ⇄ %v:%d] (%s) %d bytes sent (+%d), %d bytes received (+%d)",
```

Comment:
I think it's worth importing a bytes to human readable converter to save space and simplify `%d bytes sent` and `%d bytes received`

Something like https://github.com/cloudfoundry/bytefmt/blob/master/bytes.go 

We're probably already importing and using something similar in this repo

Reply:
👍

Reply:
indeed we're importing `github.com/dustin/go-humanize`. I'll update it the log entry to use it.